app/livechat/client/stylesheets/omnichannel.less

```diff
@@ -243,52 +243,41 @@
 	flex-direction: column;
 }
 
-.lc-chart-options {
-	// flex: 1;
-}
-
 .border-component-td {
   padding: 0.5rem;
 }
 
-.lc-analytics-chart-container {
+.omnichannel-analytics-chart-container {
 	// flex-grow: 100%;
 	flex: auto;
 	position: relative;
 	// height: inherit;
 }
 
-.lc-analytics-header {
+.omnichannel-analytics-header {
 	float: right;
-  & > .lc-date-picker-btn {
+  & > .omnichannel-date-picker-btn {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4828896)